app/assets/stylesheets/peer_reviews.scss

```diff
@@ -41,3 +41,11 @@
   padding-left: 19em;
   padding-right: 33em;
 }
+
+.peer-review-scrollbar-div {
+  padding-bottom: 8px;
+}
```

Comment:
Looking at it again though, I don't know if `scrollbar` is the best choice of noun